rules/macos/initial_access_suspicious_mac_ms_office_child_process.toml

```diff
@@ -53,31 +53,39 @@ timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-process where host.os.type == "macos" and event.type in ("start", "process_started") and
- process.parent.name:("Microsoft Word", "Microsoft PowerPoint", "Microsoft Excel") and
- process.name:
- (
-   "bash",
-   "dash",
-   "sh",
-   "tcsh",
-   "csh",
-   "zsh",
-   "ksh",
-   "fish",
-   "python*",
-   "perl*",
-   "php*",
-   "osascript",
-   "pwsh",
-   "curl",
-   "wget",
-   "cp",
-   "mv",
-   "base64",
-   "launchctl"
+process where event.action == "exec" and
+  descendant of [
+    process where event.action == "exec" and
+    process.name: (
+      "Microsoft Word",
```

Comment:
Outlook by chance?

Reply:
I can add it but its not really abused in this way.